README.md

```diff
@@ -19,3 +19,211 @@ Requires:
+```
+export GITHUB_PRIVATE_KEY="$(cat PATH_TO_PRIVATE_KEY_FILE)"
+```
+
+### Step 4.4: Create a directory containing all necessary scripts
+
+#### After [PR#10](https://github.com/EESSI/eessi-bot-software-layer/pull/10) has been merged
+
+The bot just needs one script `eessi-bot-build.slurm` available from the `scripts` directory in the repository [EESSI/eessi-bot-software-layer](https://github.com/EESSI/eessi-bot-software-layer) (or your fork of it) and an `app.cfg` that points to the location of that script on your system.
+
+First, clone the EESSI/eessi-bot-software-layer repository (or your fork of it) by running
+
+```
+git clone https://github.com/EESSI/eessi-bot-software-layer.git
+```
+
+Change directory with `cd eessi-bot-software-layer` and note the full path of the directory (`pwd`).
+
+Add a configuration file (if it doesn't exist yet) by running
+
+```
+cp -i app.cfg.example app.cfg
+```
+
+Inside `app.cfg` change the value of `build_job_script` to `FULL_PATH/scripts/eessi-bot-build.slurm`
+
+#### Before [PR#10](https://github.com/EESSI/eessi-bot-software-layer/pull/10) has been merged
+
+At the moment the bot requires scripts from the repositories [EESSI/software-layer](https://github.com/EESSI/software-layer) and [EESSI/eessi-bot-software-layer](https://github.com/EESSI/eessi-bot-software-layer) stored under a common directory (tree). Let's assume the main directory for such a directory tree would be `$USER/bot_scripts` then follow the procedure below to prepare the directory and let the bot know its location. 
+
+```
+mkdir -p $USER/bot_scripts
+mkdir -p $USER/bot_scripts/gh
+cd $USER/bot_scripts/gh
+git clone https://github.com/EESSI/software-layer
+cp software-layer/{*.sh,*.py,configure_easybuild} ..
+mkdir -p ../init
+cp software-layer/init/{eessi_environment_variables,minimal_eessi_env,eessi_software_subdir_for_host.py} ../init
+git clone https://github.com/EESSI/eessi-bot-software-layer
+cp eessi-bot-software-layer/scripts/{*.slurm,*.sh} ..
+```
+
+Inside the directory that contains the bot's python script (`eessi_bot_software_layer.py`) add the configuration file `app.cfg` and update the location of the scripts directory. That is for the above example procedure there need to be a line
+
+```
+scripts_dir = $USER/bot_scripts
+```
+
+You can obtain an example `app.cfg` from the bot's repository, e.g., assuming the above clone you could do
+
+```
+cp $USER/bot_scripts/gh/eessi-bot-software-layer/app.cfg.example PATH_TO_BOTS_PYTHON_SCRIPT/app.cfg
+```
+
+### Step 4.5: Run the EESSI bot
+
+Change directory to `eessi-bot-software-layer` (which was created by cloning the repository in Step 3 - either the original one from EESSI or your fork). Then, simply run the bot by executing
+```
+./run.sh
+```
+
+Note, if you run the bot on a frontend of a cluster with multiple frontends make sure that both the Smee client and the bot run on the same machine.
+
+The bot will log events into the console.
```

Comment:
That's actually not correct, it will only log stuff in the log file?

There's currently exactly zero output produced by the `run.sh` script (which is a bit confusing, it should at least print a message saying that the bot has started, and mention the location of the log file)

Reply:
Yep. It logs to `pyghee.log`. Changed.

Reply:
The suggested improvement of the output for when the bot starts is addressed in #9